### PR TITLE
Apply template to MedicationOrder

### PR DESCRIPTION
**Pull request by daliboz**
Applied new template to the MedicationOrder resource

### CODE REVIEW

File: content/millennium/dstu2/medications/medication-order.md [hunk 1]
```diff
@@ -7,6 +7,36 @@ title: MedicationOrder | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Medication Order resource provides orders for all medications along with administration instructions for a patient in both the inpatient and outpatient setting (orders/prescriptions filled by a pharmacy and discharge medication orders). This resource does NOT include historical or home medications documented or reported to be taken by patient, significant other or another provider.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/medications/medication-order.md]: I see Medication Order and MedicationOrder in the documentation, is one preferable?

**Comment by MaxPhilips** [comment]
+1

File: content/millennium/dstu2/medications/medication-order.md (hunk 1 above)
Comment (daliboz) [inline content/millennium/dstu2/medications/medication-order.md]: convention we've followed so far is we separate the resource name in the overview. And then reference it as the api would elsewhere. Not sure if we're stuck on that convention or not :)

Comment (cerbs) [inline content/millennium/dstu2/medications/medication-order.md]: Tense seems a little off at the end. Maybe "to have been taken ..."?

**Comment by cerbs** [comment]
+1 otherwise

**Pull request merged**